docs/earthfile/earthfile.md

```diff
@@ -106,6 +106,16 @@ Sets a value override of `<value>` for the Dockerfile build arg identified by `<
 
 In a multi-stage Dockerfile, sets the target to be used for the build. This option is similar to the `docker build --target <target-name>` option.
 
+## LOCALLY (**experimental**)
+
+#### Synopsis
+
+* `LOCALLY`
+
+#### Description
+
+The `LOCALLY` command instructs earthly to execute the following `RUN` commands locally on the host system. This feature should be used with caution as locally run commands have no guarantee they will behave the same on different systems; where possible it's recommended to use the `FROM` directive instead.
```

Comment:
We should also say that `LOCALLY` "initializes a new build environment" - the same way FROM does.

Perhaps
> The `LOCALLY` initializes a new build environment and instructs earthly to execute the following `RUN` commands locally...

Reply:
We should also say that currently only `RUN` commands are allowed after `LOCALLY`.

Reply:
It seems a bit odd saying it initialises a new environment, because it doesn't initializes anything in the context of creating a new image or snapshot. 

Maybe something like The `LOCALLY` defines a new build target in which `RUN` commands are executed locally on the host system.

Reply:
Yeah.. that's true actually. Not sure how best to say that it drops whatever build environment you have before (eg any `FROM` above `LOCALLY`).

Perhaps your original text is the best then. I wouldn't say "defines a new build target" as that's what `target:` does.

Reply:
ok, lets keep it as-is for now and perhaps change it later.